lib/ios-crash-log.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import _fs from 'fs';
+import _ from 'lodash';
+import glob from 'glob';
+import { getLogger } from 'appium-support';
+import B from 'bluebird';
+
+let fs = {
+  stat: B.promisify(_fs.statSync),
+  readFile: B.promisify(_fs.readFile)
+};
+require('date-utils');
+let logger = getLogger('ios-crash-log');
+
+class IosCrashLog { 
+  constructor() {
+    this.prevLogs = [];
+    this.logsSinceLastRequest = [];
+    this.crashDir = path.resolve(process.env.HOME, 'Library', 'Logs', 'DiagnosticReports');
+  }
+
+  async getCrashes() {
+    try {
+      let crashFiles = await glob(`${this.crashDir} /*.crash`);
+      return crashFiles;
+    } catch (err) {
+      logger.error('There was a problem getting the crash list');
```

Comment:
but we might want to just avoid the try/catch here and let the original error bubble up. what do you think @imurchie?